### PR TITLE
EO-35 add docs for deleting tests

### PR DESCRIPTION
**Pull request by gnomff**

### CODE REVIEW

File: content/api/ab-testing.apib [hunk 1]
```diff
@@ -417,6 +417,41 @@ Returns the `status`.
...
+
++ Request
+
+    + Headers
+
+            Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+            Accept: application/json
+
++ Response 200 (application/json)
+
```
Comment (awinder) [inline content/api/ab-testing.apib]: what are you thinking for 200 response here, just some simple message that says 'ab test was deleted' or something?

Comment (gnomff) [reply content/api/ab-testing.apib]: I was just going to return a 200, I guess I should look at what we normally do for deletes

Comment (gnomff) [reply content/api/ab-testing.apib]: Looks like in other situations we just return `{}`, we can do that

Comment (bizob2828) [reply content/api/ab-testing.apib]: we do 204's on deletes

File: content/api/ab-testing.apib [hunk 2]
```diff
@@ -417,6 +417,41 @@ Returns the `status`.
...
+    + Headers
+
+            Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+            Accept: application/json
+
++ Response 204 (application/json)
+
+
++ Response 404 (application/json)
```
Comment (pmcclory) [inline content/api/ab-testing.apib]: hmm - the actual behavior doesn't match this.  I just get:

```
{
  "errors": [
    {
      "message": "Resource could not be found"
    }
  ]
}
```

@awinder - I seem to remember you mentioning there's some nginx magic that sets the response body to the above for all 404s?

**Pull request merged**